torch/csrc/distributed/rpc/request_callback.cpp

```diff
@@ -9,13 +9,13 @@ namespace rpc {
 
 using namespace torch::distributed::autograd;
 
-std::shared_ptr<JitFuture> RequestCallback::operator()(Message& request) const {
+std::shared_ptr<JitFuture> RequestCallback::operator()(Message& request, std::shared_ptr<LazyStreamContext> ctx) const {
```

Comment:
nit: since we are going to access `LazyStreamContext` in general RPC files, it probably no longer belong to `tensorpipe_utils.h`. One candidate new location might be `rpc/utils.h`.